src/components/dropdownMenu/DropdownMenuItem.tsx

```diff
@@ -22,17 +23,19 @@ const DropdownMenuItem = ({
   const handleClick = useHandleDropdownMenuItemClick(rest.onClick, onSelect);
 
   return (
-    <button
-      {...rest}
-      onClick={handleClick}
-      type="button"
-      className={getDropdownMenuItemClassName(className)}
-      title={rest.title ?? rest["aria-label"]}
-    >
-      <MenuItemContent icon={icon} shortcut={shortcut}>
-        {children}
-      </MenuItemContent>
-    </button>
+    <DropdownMenuPrimitive.Item className="radix-menuitem">
```

Comment:
```suggestion
    <DropdownMenuPrimitive.Item className="radix-menu-item">
```
better readability if we separate the words via `-`